### PR TITLE
completed task5 and add some html pages

### PR DESCRIPTION
**Pull request by Inunor**

### CODE REVIEW

File: UI/js/logging.js [hunk 1]
```diff
@@ -1,58 +1,194 @@
...
+		}
+
+		removePhotoPost(id) {
+			if (photoPosts.removePhotoPost(id))
+				if (this.posts.some(post => post.id === id))
+					this.reload(0, this.posts.length - 1);
+		}
+
+		editPhotoPost(id, post) {
+			if (photoPosts.editPhotoPost(id, post)) {
```
Comment (tyndria) [inline UI/js/logging.js]: Please, use braces in case of 'if', 'for' statements

File: UI/js/logging.js [hunk 2]
```diff
@@ -1,58 +1,194 @@
...
+			this.setButtonsDisplay();
+		}
+
+		addPhotoPost(post) {
+			if (photoPosts.addPhotoPost(post))
+				this.reload(0, this.posts.length);
+		}
+
+		removePhotoPost(id) {
+			if (photoPosts.removePhotoPost(id))
```
Comment (tyndria) [inline UI/js/logging.js]: Please, use braces in case of 'if' statements

File: UI/js/logging.js [hunk 3]
```diff
@@ -1,58 +1,194 @@
...
+			let figure = document.createElement('figure');
+			figure.setAttribute('id', photoPost.id);
+
+			let img = document.createElement('img');
+			img.setAttribute('alt', 'image');
+			img.setAttribute('src', photoPost.photoLink);
+
+			let figcaption = document.createElement('figcaption');
+
+			let aLike = document.createElement('a');
```
Comment (tyndria) [inline UI/js/logging.js]: You could create function with parameters, that create element <a> with icon and pass icon's and other classes as an argument

File: UI/js/logging.js [hunk 4]
```diff
@@ -1,58 +1,194 @@
...
+			this.setButtonsDisplay();
+		}
+
+		setButtonsDisplay() {
+			let addPhotoBtn = document.getElementById('add-photo');
+			addPhotoBtn.style.display = this.isSignIn? 'block' : 'none';
+
+			var figures = document.body.querySelectorAll('figure');
+			figures.forEach(figure => {
+				figure.querySelector('.like').style.display = this.isSignIn? 'block' : 'none';
```
Comment (tyndria) [inline UI/js/logging.js]: Think how to avoid repeated code here

**Comment by tyndria** [comment]
Look, I can't get access to your functions from PhotoPostsWrapper class in console, because you don't return (export) anything from you module. See task 5: "Действия пользователя имитируются вызовами глобальных функций из консоли."

I would like to open console and call some functions with arguments and see the result in real time, so, I would like to manipulate DOM with help of your functions

File: UI/js/main.js [hunk 5]
```diff
@@ -16,21 +12,22 @@
 				photoLink: 'string',
 			};
 			this.maxLength = 200;
+			this.unchangeable = new Set(["id", "author", "createdAt", "likes"]);
```
Comment (tyndria) [inline UI/js/main.js]: Sorry, I missed this point, why field 'likes' is unchangeable?

Comment (Inunor) [reply UI/js/main.js]: In many services user can't remove 'likes' even from his post. It seemed right for me

Comment (Inunor) [reply UI/js/main.js]: I mean 'likes' from other people

**Pull request merged**